### PR TITLE
Remove day limit on previous personal events in ICal

### PR DESCRIPTION
**Pull request by AdrianAndersen**
With this change, personal events will remain in the user's personal calendar (eg. iCal, Google Calendar) [Previously, personal events disappreard from the user's personal calendar after 10 days]

### CODE REVIEW

File: lego/apps/ical/viewsets.py [hunk 1]
```diff
@@ -89,21 +89,11 @@ def personal(self, request):
...
 
         permission_handler = get_permission_handler(Meeting)
-        meetings = permission_handler.filter_queryset(
-            request.user,
-            Meeting.objects.filter(
-                end_time__gt=timezone.now()
-                - timedelta(days=constants.HISTORY_BACKWARDS_IN_DAYS)
-            ),
-        )
+        meetings = permission_handler.filter_queryset(request.user)
```
Comment (odinuge) [inline lego/apps/ical/viewsets.py]:
```suggestion
        meetings = permission_handler.filter_queryset(request.user, Meeting.objects.all())
```

File: lego/apps/ical/viewsets.py [hunk 2]
```diff
@@ -89,21 +89,11 @@ def personal(self, request):
         permission_handler = get_permission_handler(Event)
         following_events = permission_handler.filter_queryset(
             request.user,
-            Event.objects.filter(
-                followers__follower_id=request.user.id,
-                end_time__gt=timezone.now()
-                - timedelta(days=constants.HISTORY_BACKWARDS_IN_DAYS),
-            ).all(),
+            Event.objects.filter(followers__follower_id=request.user.id).all(),
```
Comment (odinuge) [inline lego/apps/ical/viewsets.py]: If `constants.HISTORY_BACKWARDS_IN_DAYS` is no longer used (I am not sure tho), it should be removed from the constants file. Just search through the repo after `HISTORY_BACKWARDS_IN_DAYS`.

Comment (odinuge) [inline lego/apps/ical/viewsets.py]:
```suggestion
            Event.objects.filter(followers__follower_id=request.user.id),
```
All isn't required.

**Comment by odinuge** [comment]
When you fix the issue I commented (see test output here: https://ci.abakus.no/webkom/lego/3603/7), the tests will run, and we will see if this breaks any of them

File: lego/apps/ical/viewsets.py (hunk 2 above)
Comment (AdrianAndersen) [reply lego/apps/ical/viewsets.py]: This constant is still used for all events, thus it should remain.

**Pull request merged**